environment.yml

```diff
@@ -1,45 +1,105 @@
-name: ising_py2
+name: ising_py3
 channels:
+  - conda-forge
```

Comment:
lmfit and others are only available from conda-forge channel - changed ising_py3 above